src/Facade/Responses/VCardEntityResponse.php

```
@@ -21,7 +21,10 @@ class VCardEntityResponse extends ETagEntityResponse
     /**
      * @return string
      */
-    public function getVCard(){
-        return isset($this->found_props['calendar-data']) ? $this->found_props['calendar-data'] : null;
+    public function getVCard() {
+        if (!isset($this->found_props['calendar-data'])) return null;
```

PHPStan (2.1.46)

[error] Method CalDAVClient\Facade\Responses\VCardEntityResponse::getVCard() should return string but returns null.


(return.type)